homeassistant/components/zha/cover.py

```diff
@@ -103,10 +111,27 @@ def async_restore_last_state(self, last_state):
 
     @property
     def is_closed(self) -> bool | None:
-        """Return if the cover is closed."""
-        if self.current_cover_position is None:
+        """Return if the cover is closed.
+
+        Consider cover closed only if both tilt and lift are 0.
+        If cover doesn't support tilt, only care about lift.
+        """
+        if self._current_position is None:
+            return None
+        return self._current_position == 0 and getattr(self, "_tilt_position", 0) == 0
```

Comment:
This change (requiring both lift and tilt to be 0) would be bad for my devices. I have 4 cover devices and after completely closing them (from a completely opened state) they all report `current_position: 0` just fine, but `current_tilt_position` is reported at various values (0, 10, 15, 20).

I'm not sure if the same problem happens for `_is_open` (requiring tilt to be 100), but I can check this if needed.

The device is the one for which I opened this issue: https://github.com/zigpy/zha-device-handlers/issues/1397

Reply:
Hm, it sounds like we need to agree on what "Closed" means.

On Ubisys J1, only if _both_ tilt and lift are 0, it means the cover is really "closed". Contrary, if `lift == 0 && tilt != 0` it means there is a physical gap between slats. In extreme case, where `lift == 0 && tilt == 100`, the slats are completely horizontal, which is a confusing state to be called "Closed".

Reply:
In general I agree that only `lift == 0 && tilt == 0` means "closed" (except for my note at the end of this comment). But it looks like my devices don't update the reported position often enough. So even though the physical slats are completely closed they might report a different value.

I don't know how it works exactly, but it looks like it only reports the changed tilt value every 1-2 seconds while it's moving. So if the last reported value was 20 and then it's fully closed (stopped) before the next update interval, it will not be updated to 0. I can get it to report 0 tilt, but only for about 30% of my tries when opening/closing it.

Also, for me 100 tilt doesn't mean "slats are horizontal". They are horizontal at 50 tilt. 100 means "closed in the other direction (while opening)". So I could even say that "closed" means `lift == 0 && (tilt == 0 || tilt == 100)` (assuming my devices would report tilt correctly).

Reply:
What an interesting window covering type. :)

What values of various setup attributes (as per ZCL spec chapter 7.4.2.1.3) are you using? Have you ever tried to execute any calibration procedure?

I have one of those Nexentro devices somewhere in my garage, but as far as I remember, the configuration app doesn’t allow any calibration. Nevertheless, my window coverings works differently (2-line system).

Reply:
> What values of various setup attributes (as per ZCL spec chapter 7.4.2.1.3) are you using?

Where do I see those?

> Have you ever tried to execute any calibration procedure?
> I have one of those Nexentro devices somewhere in my garage, but as far as I remember, the configuration app doesn’t allow any calibration.

You are correct, there is no calibration.
The shutter itself only has two hardware inputs (buttons/circuits), up and down.
The app for the Zigbee device allows me to configure the time it takes for a full lift and for a full tilt. When setting a position via HA the device calculates how long it should close the up or down circuit based on the current position and target position. While the circuits are closed it emulates lift and tilt positions based on the time the circuits are closed.

Reply:
0x0010 65535
0x001
0x001
0x001
0x001
0x001
0x001
0x001
0x001

Reply:
0x0010 65535
0x0011 0
0x0012 65535
0x0013 0
0x0014 None
0x0015 None
0x0016 None
0x0017 0
0x0018 None
0x0019 None

Reply:
I have the same Nexentro actuators - and see similar  issues as @cremor  (non zero tilt if closed).  
However my personal impression is this comes from Apple HomeKit/Siri.  (What I use mostly to operate my window blinds)  The simple "Hey Siri  Jaousie im Bad zu/auf" seems to send just set_cover_position  (see Pic) 
<img width="751" alt="image" src="https://github.com/home-assistant/core/assets/16188530/ae59fe9c-60eb-4a4e-b7a9-4ae2016eabb5">
<img width="731" alt="image" src="https://github.com/home-assistant/core/assets/16188530/c48a06c6-51b4-4793-aa95-d9dd32314227">
what ends in a non zero tilt:
<img width="871" alt="image" src="https://github.com/home-assistant/core/assets/16188530/454f9cda-fc8b-45bc-ab0f-1d8f262fb7ce">
if I do the same in HA with cover.close_cover:
<img width="603" alt="image" src="https://github.com/home-assistant/core/assets/16188530/0b5ce8cc-f4d9-487b-b370-7d5137d3c6b4">
<img width="909" alt="image" src="https://github.com/home-assistant/core/assets/16188530/3a2c8a71-c666-4db1-aca4-aa1f5354854f">
Also closing the blind/Nexentro device via the wall button has the same   effect  pos=0 tilt=0 
imho:  HA nexentro has no problem - it Siri - she need to lear tilting blinds

Reply:
I tested it with cover.close_cover and the physical buttons and still got tilt values > 0.